DevAAC/Models/Player.php

```diff
@@ -286,4 +286,15 @@ public function setLastipAttribute($longip)
     {
         $this->attributes['lastip'] = chbo(ip2long($longip));
     }
+
+    public function getTownAttribute()
+    {
+        if(defined('TOWNS'))
```

Comment:
Table towns exists in database since https://github.com/otland/forgottenserver/pull/2386